.github/workflows/ci.yml

```diff
@@ -101,3 +101,30 @@ jobs:
       - name: Teardown integration environment
         run: npm run testenv:stop
         if: always()
+
+  build-publish:
+    runs-on: ubuntu-latest
+    needs:
+      - lint
+      - test
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          token: ${{ secrets.GH_TOKEN || secrets.GITHUB_TOKEN }}
+          fetch-depth: 0
```

Comment:
This will fetch the entire history, which is a bit excessive. Would recommend removing.